source/grid/manifold_lib.cc

```diff
@@ -1914,8 +1914,8 @@ ::compute_chart_points (const ArrayView<const Point<spacedim>> &surrounding_poin
         // enable the structdim 2 optimization
         use_structdim_2_guesses = true;
       else if (spacedim == 3)
-        // otherwise these vectors are roughly orthogonal are roughly
-        // orthogonal: enable the structdim 3 optimization if we are in 3D
+        // otherwise these vectors orthogonal are roughly orthogonal: enable
```

Comment:
I think that there's still an extra "orthogonal" in this sentence.

Reply:
You're right. Fixed.